frontend-react/src/components/Table/Pagination.tsx

```diff
@@ -0,0 +1,8 @@
+export const OVERFLOW_INDICATOR = Symbol("...");
+export type SlotItem = number | typeof OVERFLOW_INDICATOR;
+
+export interface PaginationProps {
+    slots: SlotItem[];
+    setCurrentPage: (pageNum: number) => void;
+    currentPageNum: number;
```

Comment:
This is setup for the UI, work, yes?

Reply:
Correct. The values in this module are needed by the hook but should be owned by the UI component, so I added them here.